spec/helpers/mailer_helper_spec.rb

```diff
@@ -1,13 +1,76 @@
 require "spec_helper"
 
 describe MailerHelper do
+  let(:work) { create(:work) }
+  let(:chapter) { create(:chapter) }
+  let(:series) { create(:series) }
+
   describe "style_creation_link" do
     it "nests red link inside bold inside italics" do
-      work = create(:work)
       expect(style_creation_link(work.title, work_url(work))).to eq("<i><b><a style=\"color:#990000\" href=\"#{work_url(work)}\">#{work.title}</a></b></i>")
     end
   end
 
+  describe "#creation_word_count" do
```

Comment:
I think we can skip these specs - they only make sure that `I18n.t` works. We're also covered by the `creation_*_with_word_count` specs.